starboard/android/apk/app/src/main/java/dev/cobalt/coat/CobaltActivity.java

```diff
@@ -421,4 +421,18 @@ public void onLowMemory() {
   public long getAppStartTimestamp() {
     return timeInNanoseconds;
   }
+
+  public void setPreferMinimalPostProcessing(boolean value) {
+    Runnable runnable =
+        new Runnable() {
+          @Override
+          public void run() {
+            if (getDisplay().isMinimalPostProcessingSupported()) {
```

Comment:
one thing we should doublecheck: What android API version is this available of in ? There may need to be an extra `if android_version` clause here. We have a few other examples of those in Starboard code IIRC

Reply:
Done. Both APIs were added in API 30, added a check to return early before doing the UI-thread work.

Reply:
lgtm